ts/features/bonus/siciliaVola/saga/networking/handleGetAereoportiBeneficiario.ts

```diff
@@ -0,0 +1,69 @@
+import { ActionType } from "typesafe-actions";
+import { call, put } from "redux-saga/effects";
+import { BackendSiciliaVolaClient } from "../../api/backendSiciliaVola";
+import { getGenericError, getNetworkError } from "../../../../../utils/errors";
+import { SagaCallReturnType } from "../../../../../types/utils";
+import { isDefined } from "../../../../../utils/guards";
+import { SessionManager } from "../../../../../utils/SessionManager";
+import { MitVoucherToken } from "../../../../../../definitions/io_sicilia_vola_token/MitVoucherToken";
+import { svGenerateVoucherAvailableDestination } from "../../store/actions/voucherGeneration";
+import { AvailableDestinations } from "../../types/SvVoucherRequest";
+import { AeroportoSedeBeanList } from "../../../../../../definitions/api_sicilia_vola/AeroportoSedeBeanList";
+
+// TODO: add the mapping when the swagger will be fixed
+const mapKinds: Record<number, string> = {};
+
+// convert a success response to the logical app representation of it
+const convertSuccess = (
+  availableDestinations: AeroportoSedeBeanList
+): AvailableDestinations =>
+  availableDestinations
+    .map(d => (d.denominazione ? d.denominazione : undefined))
+    .filter(isDefined);
+
+export function* handleGetAereoportiBeneficiario(
```

Comment:
Aeroporti :-)

Yep, I also need to check [this](https://accademiadellacrusca.it/it/consulenza/aereo-ma-aeroporto-perch%C3%A9/76) from time to time...

Reply:
Thank you :) Fixed in [0f3c165](https://github.com/pagopa/io-app/pull/3267/commits/0f3c165c67d51bfcc2dbae3e266acf79277073da)